src/main/java/org/jenkinsci/plugins/workflow/pipelinegraphanalysis/GenericStatus.java

```diff
@@ -46,6 +46,11 @@
     UNSTABLE,
 
     /**
+     * Not complete: still executing, in a node block, but the node block is in queue.
+     */
+    QUEUED,
```

Comment:
Yeah, I don't love it, but the idea here is that a parallel with no `IN_PROGRESS` branches (i.e., because all but one of them are complete) and one `QUEUED` branch (because, well, it's queued) should show up as `QUEUED`, but if there's even one actually `IN_PROGRESS` branch, that should trump the `QUEUED`. cc @i386 for the behavior logic. =)